src/Microsoft.AspNetCore.Mvc.Core/project.json

```diff
@@ -51,12 +51,7 @@
     "System.Diagnostics.DiagnosticSource": "4.0.0-*"
   },
   "frameworks": {
-    "net451": {
-      "frameworkAssemblies": {
-        "System.Reflection": "",
-        "System.Runtime": ""
-      }
-    },
+    "net451": { },
```

Comment:
We tried that change - https://github.com/aspnet/DependencyInjection/commit/99f4f17a88277c4edfabd988ae994db79c1b17a7 - but ended up reverting it because of odd CLI issues + issues with restore. I'll work on bringing it back soon.